sdks/go/pkg/beam/core/runtime/xlangx/expand.go

```diff
@@ -20,25 +20,40 @@ import (
 
 	"github.com/apache/beam/sdks/go/pkg/beam/internal/errors"
 	jobpb "github.com/apache/beam/sdks/go/pkg/beam/model/jobmanagement_v1"
+	"github.com/apache/beam/sdks/go/pkg/beam/model/pipeline_v1"
 	"google.golang.org/grpc"
 )
 
 // Expand queries the expansion service to resolve the ExpansionRequest
```

Comment:
We might want to expand the description here, since the expansion request is being created "in house".

// Expand submits the pipeline components and root transform to be expanded by the given expansion service.
//
// Most should call beam.CrossLanguage to access foreign transforms rather than calling this function directly.

The latter segment is probably unnecessary since it won't add things to the pipeline properly.. but we don't currently have too much documentation.

Reply:
Done.